cli/cobra.go

```diff
@@ -360,6 +376,9 @@ Invalid Plugins:
 
 Run '{{.CommandPath}} COMMAND --help' for more information on a command.
 {{- end}}
+{{- if displayHelpLink .}}
+{{ cyan "To get more help with docker, check out guides at https://docs.docker.com/go/guides/" }}
```

Comment:
Wondering (discussed that option briefly with @chris-crone) if instead of hard-coding, we should use annotations similar to; https://github.com/docker/cli/blob/v20.10.0-rc1/cli/cobra.go#L195-L198

That way we could just print the information if its set on a specific (or "any") command

Reply:
+1 to this but we can also do it as a follow up if we add links for specific helps

Reply:
Right, added the additional help message in root annotation, so we can add specific messages to any command as we like in the future. (also renamed methods for something more homogeneous